engine/src/main/java/org/kigalisim/engine/SingleThreadEngine.java

```diff
@@ -27,6 +28,16 @@
  * <p>This implementation provides the core simulation engine functionality
  * without thread safety considerations. It manages substance streams, equipment
  * populations, and various calculations related to the Montreal Protocol simulation.</p>
+ * 
+ * <p>Key differences from JavaScript Engine implementation:
```

Comment:
Thank you for this! However, I think I will document this elsewhere. Please remove this new paragraph (`<p>`)